components/infobox/wikis/starcraft2/infobox_person_player_custom.lua

```diff
@@ -418,7 +423,7 @@ function CustomPlayer._getEarningsMedalsData(player)
 end
 
 function CustomPlayer._addPlacementToEarnings(earnings, earnings_total, data)
-	local mode = _EARNING_MODES[(data.players or {}).type or ''] or 'other'
+	local mode = _EARNING_MODES[data.opponenttype or ''] or DEFAULT_EARNINGS_MODE
```

Comment:
```suggestion
	local mode = _EARNING_MODES[data.opponenttype] or DEFAULT_EARNINGS_MODE
```
Should be OK

Reply:
yeah award place3ments get there, so the `or ''` is needed untill all that data has been purged
(i updated the module used to store awards some days ago but there certainly is still data that would cause issues)